wsrep_thread_service.h

```diff
@@ -0,0 +1,349 @@
+/*
+ * Copyright (C) 2019 Codership Oy <info@codership.com>
+ *
+ * This file is part of wsrep-API.
+ *
+ * Wsrep-API is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
```

Comment:
If we want to have a separate wsrep_loader.h file, it makes sense to put the same wording in there. And probably in wsrep_api.h as well. I mean, to have a common head in all three files.

Reply:
Maybe better to update the license header for all files in separate PR.